core/main_test.ts

```diff
@@ -2609,6 +2611,79 @@ SELECT 2`
         });
       });
     });
+
+    suite("incremental table on schema change", () => {
```

Comment:
It's a bit overkill to test all of these exhaustively, I would just add one of the (non-default) options to the action configs and SQLX tests. The other tests include the defaults.

Where I'm suggesting adding:
* https://github.com/dataform-co/dataform/blob/c799ec73800f23d0cbf03e1862a4a841e79b1b6d/core/main_test.ts#L2994, and
* https://github.com/dataform-co/dataform/blob/c799ec73800f23d0cbf03e1862a4a841e79b1b6d/core/main_test.ts#L2373

Reply:
Followed your suggestion remove additional tests and add some non-default values to the places you suggest. I also make sure that the whole mapper is properly covered.